### PR TITLE
formik/ENG-5159 remove redux form on ContentsQueryConfig

### PR DESCRIPTION
**Pull request by ryanjpburgos**

### CODE REVIEW

File: src/helpers/getAppBuilderWidgetForm.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ const cmsWidgetForms = {
     body: SingleContentConfigBody,
   },
   listViewerConfig: {
-    default: ContentsQueryConfigContainer,
-    body: ContentsQueryFormBody,
+    default: formBody,
+    body: formBody,
   },
```
Comment (ichalagashvili) [inline src/helpers/getAppBuilderWidgetForm.js]: I wonder why did you need to make this change?

File: src/ui/widget-forms/ContentsQueryConfigContainer.js [hunk 2]
```diff
@@ -27,28 +26,35 @@ import { NoDefaultWarningModalId } from 'ui/widget-forms/publish-single-content-
 const nopage = { page: 1, pageSize: 0 };
 
 export const mapStateToProps = (state, ownProps) => {
-  const formToUse = get(ownProps, 'extFormName', ContentsQueryContainerId);
   const parentField = get(ownProps, 'input.name', '');
```
Comment (ichalagashvili) [inline src/ui/widget-forms/ContentsQueryConfigContainer.js]: please write `field.name` instead of `input.name` since formik has `field` not `input`.

File: src/ui/widget-forms/ContentsQueryConfigContainer.js [hunk 3]
```diff
@@ -27,28 +26,35 @@ import { NoDefaultWarningModalId } from 'ui/widget-forms/publish-single-content-
...
-    selectedCategories: formValueSelector(formToUse)(state, putPrefixField('categories')),
-    selectedInclusiveOr: formValueSelector(formToUse)(state, putPrefixField('orClauseCategoryFilter')),
-    widgetConfigFormData: getFormValues(formToUse)(state),
     defaultLanguageCode: getDefaultLanguage(state),
   };
 };
 
 export const mapDispatchToProps = (dispatch, ownProps) => {
-  const formToUse = get(ownProps, 'extFormName', ContentsQueryContainerId);
   const parentField = get(ownProps, 'input.name', '');
```
Comment (ichalagashvili) [inline src/ui/widget-forms/ContentsQueryConfigContainer.js]: please write `field.name` instead of `input.name` since formik has field not input.

**Review by ichalagashvili — COMMENTED** [review]
Code reviewed and few comments, now starting manually testing the changes locally.

File: src/helpers/getAppBuilderWidgetForm.js (hunk 1 above)
Comment (ryanjpburgos) [reply src/helpers/getAppBuilderWidgetForm.js]: I removed from this file src/ui/widget-forms/ContentsQueryConfigContainer.js the other export, I think that it's a duplication, I don't know why we have the default and the body

File: src/ui/widget-forms/ContentsQueryConfigContainer.js (hunk 2 above)
Comment (ryanjpburgos) [reply src/ui/widget-forms/ContentsQueryConfigContainer.js]: done

File: src/ui/widget-forms/ContentsQueryConfigContainer.js (hunk 3 above)
Comment (ryanjpburgos) [reply src/ui/widget-forms/ContentsQueryConfigContainer.js]: done

**Review by ichalagashvili — APPROVED** [review]
